documentation/CHANGELOG_TEMPLATE.md

```diff
@@ -0,0 +1,40 @@
+# Changelog API Sub Project name
+
+<!--NOTE: in above title replace "API Sub Project name" with actual API Sub Project name and remove this line-->
```

Comment:
With the changes coming with https://github.com/camaraproject/Governance/issues/142 there could be multiple repositories per Sub Project. 

```suggestion
# Changelog §API Repository Name§

<!--NOTE: in above title replace §API Repository Name§ with the actual repository name and remove this comment-->
```